File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
#include <iostream>
using namespace std;

int main() {
	
	int n;
	cin >> n;

	char number[100];
	int sum = 0;

	for (int i = 0; i < n; i++)
	{
		cin >> number[i];
		sum += (int)number[i] - 48;
	}
	
	cout << sum << endl;
	
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        sc.nextLine();
        String number = sc.nextLine();
        sc.close();

        int sum = 0;

        for (int i = 0; i < n; i++) {
            sum += number.charAt(i) - 48;
        }
        System.out.println(sum);
    }
}